### PR TITLE
Namespace List notes

### LINKED ISSUE #1678: Document Namespaces in WebUI and sensuctl
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Steps to Reproduce (for bugs)

1.
2.
3.

## Context

New feature for 5.12, Namespaces listed in both the webUI and sensuctl are now limited to those which the user has access to. 

* Browser(s) and version(s): 
* OS:

### PR DESCRIPTION
**Pull request by Blue0ctober**
added note on `sensuctl namespace list` and UI

## Description

Adds additional context around listing namespaces with non-admin users.

## Motivation and Context

Closes #1678

### CODE REVIEW

File: content/sensu-go/5.12/reference/rbac.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ You can use [sensuctl][2] to view all namespaces within Sensu:
 sensuctl namespace list
 {{< /highlight >}}
 
+_Note: If logged in with a non Admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```
Comment (rgeniesse) [inline content/sensu-go/5.12/reference/rbac.md]:
```suggestion
_Note: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```

I see other areas of the docs site use this format. Or we can get rid of it entirely and say something like "If logged with with a user that does not have administrative access, ..."

**Review by rgeniesse — CHANGES_REQUESTED** [review]
See suggestion inline.

File: content/sensu-go/5.12/reference/rbac.md [hunk 2]
```diff
@@ -48,6 +48,8 @@ You can use [sensuctl][2] to view all namespaces within Sensu:
 sensuctl namespace list
 {{< /highlight >}}
 
+_Note: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```
Comment (Blue0ctober) [inline content/sensu-go/5.12/reference/rbac.md]:
```suggestion
_NOTE: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```

File: content/sensu-go/5.12/dashboard/overview.md [hunk 3]
```diff
@@ -43,6 +43,7 @@ See the [role-based access control reference][3] for [default user credentials][
 The dashboard displays events, entities, checks, and silences for a single namespace at a time.
 By default, the dashboard displays the `default` namespace.
 To switch namespaces, select the menu icon in the upper-left corner, and choose a namespace from the dropdown.
+The namespaces listed will be limited to those that the user has access to.
```
Comment (cwjohnston) [inline content/sensu-go/5.12/dashboard/overview.md]:
```suggestion
**LICENSED TIER**: The namespaces listed will be limited to those that the current user has access to.
```

File: content/sensu-go/5.12/reference/rbac.md [hunk 4]
```diff
@@ -48,6 +48,8 @@ You can use [sensuctl][2] to view all namespaces within Sensu:
 sensuctl namespace list
 {{< /highlight >}}
 
+_NOTE: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```
Comment (cwjohnston) [inline content/sensu-go/5.12/reference/rbac.md]:
```suggestion
_NOTE: For licensed tier users,`sensuctl namespace list` will list only the namespaces that the current user has access to._
```

File: content/sensu-go/5.12/reference/rbac.md [hunk 5]
```diff
@@ -682,7 +684,7 @@ spec:
 
 ## Role bindings and cluster role bindings
 
-A **role binding** assigns a **role** or **cluster role** to users and groups within a namesapce.
+A **role binding** assigns a **role** or **cluster role** to users and groups within a namespace.
```
Comment (cwjohnston) [inline content/sensu-go/5.12/reference/rbac.md:687]: Good catch! 👍

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**